collections/_article/release-candidate-godot-4-3-rc-3.md

```diff
@@ -0,0 +1,94 @@
+- Input: X11: Use motion event button state instead of async state ([GH-95008](https://github.com/godotengine/godot/pull/95008)).
+- Input: Windows: Reject `WM_POINTER(UP/DOWN)` messages for non pen pointer type ([GH-95155](https://github.com/godotengine/godot/pull/95155)).
+- Multiplayer: ENet: Better handle disconnected peers in DTLS server ([GH-95067](https://github.com/godotengine/godot/pull/95067)).
+- Multiplayer: Avoid error spam in relay protocol when clients disconnect ([GH-95192](https://github.com/godotengine/godot/pull/95192)).
+- Multiplayer: Fix relay protocol routing with negative targets ([GH-95194](https://github.com/godotengine/godot/pull/95194)).
+- Network: WS: Fix `set_no_delay` on Windows ([GH-95233](https://github.com/godotengine/godot/pull/95233)).
+- Porting: Windows: Check if transparency is enabled in the project setting before applying DWM blur ([GH-95009](https://github.com/godotengine/godot/pull/95009)).
+- Porting: macOS: Attempt to terminate process normally before using `forceTerminate` ([GH-95191](https://github.com/godotengine/godot/pull/95191)).
+- Porting: macOS: Load `OpenGL.framework` by path to avoid issues with non-Latin executable names ([GH-95235](https://github.com/godotengine/godot/pull/95235)).
+- Rendering: D3D12: Avoid cases of redundant render target clears ([GH-95064](https://github.com/godotengine/godot/pull/95064)).
+- Rendering: D3D12: Avoid crash on exit ([GH-95074](https://github.com/godotengine/godot/pull/95074)).
+- Rendering: Fix LightmapGI causes crash when using `--headless` ([GH-95103](https://github.com/godotengine/godot/pull/95103)).
+- Shaders: VisualShader: Reduce size changes of nodes when connecting/disconnecting ([GH-95061](https://github.com/godotengine/godot/pull/95061)).
+
+## Changelog
+
+**18 contributors** submitted **40 improvements** for this new snapshot. See our [**interactive changelog**](https://godotengine.github.io/godot-interactive-changelog/#4.3-rc3) for the complete list of changes since the 4.3-rc2 snapshot. You can also review [all changes included in 4.3](https://godotengine.github.io/godot-interactive-changelog/#4.3) compared to the previous 4.2 feature release.
+
+This release is built from commit [`03afb92ef`](https://github.com/godotengine/godot/commit/03afb92efa18874da19f7fc185a32c005d20aa1d).
+
+## Downloads
+
+{% include articles/download_card.html version="4.3" release="rc3" article=page %}
+
+**Standard build** includes support for GDScript and GDExtension.
+
+**.NET build** (marked as `mono`) includes support for C#, as well as GDScript and GDExtension.
+- See also [C# platform support](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/index.html#c-platform-support).
+
+If you want to test the new Windows ARM64 builds, you will now find them listed in the usual download page.
```

Comment:
This bit depends on merging https://github.com/godotengine/godot-website/pull/885 first.